package.json

```
@@ -23,10 +23,7 @@
     "test14": "#",
     "test15": "#",
     "test16": "#",
-    "wut": "lerna run --scope wix-ui-tpa --stream",
-    "start:wut": "npm run wut -- start",
-    "test:wut": "npm run wut -- test",
-    "generate:wut": "npm run wut -- generate"
+    "wix-ui-tpa": "./run-package-command.sh wix-ui-tpa"
```

maybe?
```suggestion
    "wut": "./run-package-command.sh wix-ui-tpa"
```
Less to type 😁

---

only for those who don't have autocomplete in the terminal :)

which is probably the vast majority :(
it can be `wut`, i'm just thinking for future when we have more packages if that wont become confusing. you know, `wsr`, `wut`, `wuf`, `wbu`, `wuc` etc. maybe too many abbreviations?

---

I think it would be easier to use `wut`.
@sivanharel @nitayneeman what do you think?

---

let's have `wut` until further notice, let's not delay :)